_posts/2021-01-18-getting-started-kafka.adoc

```diff
@@ -0,0 +1,303 @@
+---
+layout: post
+title: Getting Started with Apache Kafka in your Quarkus application
+date: 2021-01-13 #TODO Update this date
+tags: kafka
+synopsis: How to build a Quarkus application using Apache Kafka in less than 10 minutes
+author: cescoffier
+---
+
+https://kafka.apache.org/[Apache] Kafka is a prevalent distributed streaming platform offering a unique set of characteristics such as message retention, replay capabilities, consumer groups, and so on.
+Kafka is highly scalable, fault-tolerant, and is becoming the spine of many modern systems.
+That being said, Kafka is not the only one out there, and choosing the right messaging system for your system can be challenging.
```

Comment:
Is there a way you can remove one of the "system"s?

"messaging system for your system" sounds a bit weird

Reply:
oh yeah. Switching to "right messaging technology for your application can be challenging."